crates/goose/src/prompts/system.md

```diff
@@ -10,12 +10,21 @@ to interactions with sytems that are not currently active. The currently
 active systems are below. Each of these systems provides tools that are
 in your tool specification.
 
+When a system is added that supports Resources,, we add two tools,
```

Comment:
nit: i think we can just delete this entirely. the tool descriptions and the below hint per system should cover this i think!